src/components/VideoCapture/index.tsx

```diff
@@ -80,6 +80,10 @@ const RECORDING_TIMEOUT_ERRORS_MAP: Record<
     name: 'FACE_VIDEO_TIMEOUT',
     type: 'warning',
   },
+  data: {
```

Comment:
This is triggering an analytics event (from inside `<CameraError>`), please add this to the `trackerData.tsx` file otherwise the event won't be send to our in house analytics.